install

```diff
@@ -193,10 +202,9 @@ group_chmods = %w[ bin etc Frameworks include lib sbin share var
                .select { |d| chmod?(d) }
 # zsh refuses to read from these directories if group writable
 zsh_dirs = %w[share/zsh share/zsh/site-functions]
-user_chmods = zsh_dirs
-              .map { |d| File.join(HOMEBREW_PREFIX, d) }
-              .select { |d| chmod?(d) }
-chmods = group_chmods + user_chmods
+           .map { |d| File.join(HOMEBREW_PREFIX, d) }
+user_only_chmods = zsh_dirs.select { |d| user_only_chmod?(d) }
```

Comment:
I think you can omit changing the variable and messaging name for this; people don't need to know/care about ZSH's special case.